whitelist/whitelist.go

```diff
@@ -0,0 +1,73 @@
+// Package whitelist provides methods for managing an app's whitelisted IP's.
+package whitelist
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+
+	deis "github.com/deis/controller-sdk-go"
+	"github.com/deis/controller-sdk-go/api"
+)
+
+// List IP's whitelisted for an app.
+func List(c *deis.Client, appID string) (api.Whitelist, error) {
```

Comment:
I'd just use introduce a function called appsettings.Get that returns an appsettings object then you can do w/e you want with it in whitelist:list. That way we don't have duplicate code everywhere